### ISSUE TITLE
Add target commands to Lucifer, Cerberus and Desolator

### ISSUE THREAD

**Issue by Lasica** [ZeroK-RTS/Zero-K#4582]
They are useful, especially with addition of ctrl + target to hit towards enemy if idle. Because of cloak/owl radar jamming sometimes you want to make them shoot at certain spot if nothing else is visible.

I had few games already where I've lacked it a lot. I've built a Lucifer next to Aspis forest and wanted it to fire at their edge to deplete the shields, but it backfired when enemy appeared and I forgot to cancel the order to force fire.

I don't see the reason why it's not added for the turrets.

**Comment by sprunk** [comment]
> I had few games already where I've lacked it a lot. I've built a Lucifer next to Aspis forest and wanted it to fire at their edge to deplete the shields, but it backfired when enemy appeared and I forgot to cancel the order to force fire.

Set-target would suffer from the same issue I think. I don't think there is any difference between force-fire and set-target for units that can't move.

**Comment by Lasica** [comment]
You are wrong. Set target makes prioritize unit when units are in range. Force fire makes the unit fire at designated spot ignoring all else. See Stardust behavior for example.

**Comment by sprunk** [comment]
Sounds good.
